src/xdocs/google_style.xml

```diff
@@ -799,17 +799,21 @@
                 <td>
                   <span class="wrapper inline">
                     <img
-                        src="images/ok_blue.png"
+                        src="images/ok_green.png"
                         alt="" />
                   </span>
                   <a href="checks/sizes/linelength.html#LineLength">LineLength</a>
                   <br />
-                  We can detect URL with protocol type as http://, https:// etc.
```

Comment:
This is not related to JSNI. Why did you remove it? Could you recheck please if we cover URLs in tests and update them if it is missed.

Reply:
I accidentally removed it. But I checked about it, we have support to ignore URLs:

https://github.com/checkstyle/checkstyle/blob/173925e9a52370dd80e66b2e847e1b0ff1b1a7eb/src/main/resources/google_checks.xml#L45-L49

But there is no tests for it. I'll update add tests soon. Thanks for pointing this out :)

Reply:
> I accidentally removed it

I still see the line is removed, please put it back.